alf/networks/value_networks.py

```diff
@@ -23,25 +23,6 @@
 from alf.tensor_specs import TensorSpec
 
```

Comment:
Does this file also need to include
`from alf.layers import _create_lstm_cell_state_spec`
?

Reply:
layers._create_lstm_cell_state_spec is what I used in this file

Reply:
Ah, you are right.